docs/conceptual/storage-format.rst

```diff
@@ -5,7 +5,7 @@
 .. _storage-format:
 
 ********************************
- Storage format
+ Storage formats
```

Comment:
if they only use one storage format (which they seem to), this should remain singular

Reply:
The original yml file in develop has 'storage formats', so made changes in the latest version to reflect that.

https://github.com/Rmalavally/hipSPARSELt/blob/develop/docs/sphinx/_toc.yml.in

Singular makes sense.

Reply:
> The original yml file in develop has 'storage formats', so made changes in the latest version to reflect that.
> 
> https://github.com/Rmalavally/hipSPARSELt/blob/develop/docs/sphinx/_toc.yml.in
> 
> Singular makes sense.

 >  - file: reference/storage-format.rst
      title: Storage formats